src/dsl/openapi-builder30.ts

```diff
@@ -45,8 +45,19 @@ export class OpenApiBuilder {
     ): string {
         return JSON.stringify(this.rootDoc, replacer, space);
     }
-    getSpecAsYaml(): string {
-        return yaml.stringify(this.rootDoc);
+    getSpecAsYaml(
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        replacer?: any[] | ((key: any, value: any) => unknown) | null,
+        options?:
+            | string
+            | number
+            | (yaml.DocumentOptions &
+                  yaml.SchemaOptions &
+                  yaml.ParseOptions &
+                  yaml.CreateNodeOptions &
+                  yaml.ToStringOptions)
```

Comment:
I'd suggest to make a reference to the types of the `yaml.stringify()` parameters instead

Reply:
```suggestion
        replacer?: Parameters<typeof yaml.stringify>[1],
        options?: Parameters<typeof yaml.stringify>[2]
```
then you'd not need:
- to reproduce internal typing of another module
- to mute ESLint error

That will also make it future resistant if `yaml` dependency upgrades, @urugator

Reply:
I wanted to do that, but it's is not doable for overloaded functions, because TS doesn't know which overload to pick.

Reply:
pick the one having `replacer`, consider `Extract` for that, @urugator

Reply:
Dunno what you mean with the `Extract`, but turns out `Parameters` are parameters of the last signature, which is the one  we need. So unless they swap or introduce new signatures it should be fine I guess.